lib/core/versions/latest/CreateOperation.ts

```diff
@@ -12,11 +12,11 @@ import SidetreeError from '../../../common/SidetreeError';
 interface SuffixDataModel {
   operationDataHash: string;
   recoveryKey: PublicKeyModel;
-  nextRecoveryOtpHash: string;
+  nextRecoveryCommitmentHash: string;
```

Comment:
Discussion: not blocking check in, but I thought I would be fine with "CommitmentHash" but the more I think about it the more I think it should be just "Commitment", one of the reason is that if we ever change the way we compute the commitment, then calling it a hash would not be correct.